packages/web-components/src/calendar/calendar.styles.ts

```diff
@@ -148,14 +150,59 @@ ${display("inline-block")} :host {
 `.withBehaviors(
   forcedColorsStylesheetBehavior(
       css`
-          .day.selected {
-              color: ${SystemColors.Highlight};
-          }
-
-          .today .date {
-              background: ${SystemColors.Highlight};
-              color: ${SystemColors.HighlightText};
-          }
+        :host {
+          forced-color-adjust: auto;
+          background: ${SystemColors.ButtonFace};
+          outline: 1px solid ${SystemColors.CanvasText};
+        }
+        .day,
+        .interact .day,
+        .week-day {
+            background: ${SystemColors.Canvas};
+            color: ${SystemColors.CanvasText};
+            fill: currentcolor;
+        }
+        .interact .day:not(.disabled, .inactive):hover {
+            background: ${SystemColors.ButtonFace};
+            color: ${SystemColors.ButtonText};
+            outline: 1px solid ${SystemColors.Highlight};
+        }
+        .week-day:${focusVisible},
+        .interact .day:not(.disabled, .inactive):${focusVisible} {
+            forced-color-adjust: none;
+            background: :${SystemColors.ButtonFace};
+            border-color: transparent;
+            box-shadow: 0 0 0 calc(${focusStrokeWidth} * 1px) ${SystemColors.Highlight} inset;
```

Comment:
Let's look again at the use of `outline` and `box-shadow` for focus state. With one of my recent PRs the focus state should be through two helpers now. For this component there's space to use the normal treatment `focusTreatmentBase`. Just set `outline-color` in HC styles.

Reply:
removed box-shadow and outline and used `focusTreatmentBase`